server/lib/get-browsers.js

```diff
@@ -13,9 +13,11 @@ let updated = statSync(join(ROOT, '../../../pnpm-lock.yaml')).mtime
 
 export async function getBrowsers(query, region) {
   let browsersByQuery = []
+  let preparedQuery
 
   try {
-    browsersByQuery = browserslist(query)
+    preparedQuery = prepareQuery(query)
```

Comment:
Not sure that this line should be in `try`.

I dont see transformations in `prepareQuery()` which should be replaced to `Unknown browser query` error.

Reply:
Fair enough. Moved it out of the `try` block.
Not sure if I need to do the preparation somewhere before, i.e., on HTTP-handler, where the "raw" query is extracted from the URL.